plugins/experimental/traffic_dump/traffic_dump.cc

```diff
@@ -32,22 +32,65 @@
 #include <arpa/inet.h>
 #include <netinet/in.h>
 
+#include <algorithm>
 #include <sstream>
 #include <iomanip>
 #include <chrono>
 #include <atomic>
 #include <string>
 #include <string_view>
+#include <unordered_set>
 
 #include "tscore/ts_file.h"
+#include "tscpp/util/TextView.h"
 #include "ts/ts.h"
 
 namespace
 {
 const char *PLUGIN_NAME   = "traffic_dump";
 const std::string closing = "]}]}";
+std::string defaut_sensitive_field_value;
+
+// A case-insensitive comparitor used for comparing HTTP field names.
+struct InsensitiveCompare {
```

Comment:
Why not use `strcasecmp`? It's overloaded for `string_view`
https://github.com/SolidWallOfCode/trafficserver/blob/master/include/tscpp/util/TextView.h#L48

Reply:
Yeah, that simplifies things.